src/modules/song/types.ts

```diff
@@ -297,3 +299,22 @@ export interface ProcessStreamTokenAgreementRequest {
   accepted: boolean;
   songId: string;
 }
+
+export interface CreateMintSongPaymentRequest {
+  readonly songId: string;
+  readonly changeAddress: string;
+  readonly utxoCborHexList: ReadonlyArray<string>;
+}
+
+export interface CborHexRequest {
+  readonly cborHex: string;
+}
+
+export interface CborHexResponse {
```

Comment:
as part of the `getSongMintPayment` comment above, we also need to remove `GetSongMintPaymentResponse` after master is merged here